src/resources/discordTimestamps.md

```diff
@@ -0,0 +1,33 @@
+# Discord Timestamps
+Discord timestamps are used to provide time in multiple formants. The information is given according to the user's timezone and locale. Discord timestamps are built with the Unix Time system, meaning that they are dynamic. These can be used by anyone; This includes users, webhooks, and applications.

```

Comment:
```suggestion
Discord timestamps are used to provide time in multiple formats. The information is given according to the user's timezone and locale. Discord timestamps are built with the Unix Time system, meaning that they are dynamic. These can be used by anyone; This includes users, webhooks, and applications.
```

Reply:
Also, in my opinion, the description could have more better explanation that the current one tho.